projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -698,7 +699,11 @@ class="screen-reader-text"
 							),
 							esc_attr( $data['subscribe_placeholder'] ),
 							esc_attr( $data['subscribe_email'] ),
-							esc_attr( $subscribe_field_id )
+							esc_attr( $subscribe_field_id ),
+							( ! empty( $data['subscribe_email'] )
+								? 'disabled="disabled" title="' . esc_attr__( "You're logged in with this email", 'jetpack' ) . '"'
+								: ''
+							)
```

Comment:
Nit: Just `disabled` instead of `disabled="disabled"` should be enough.

Should we wait for translations to appear before merging this?

Reply:
> Should we wait for translations to appear before merging this?

Not possible in Jetpack repo sadly. :-(

We could wrap in check if the translation exists, just don't recall now how to do that in Jetpack backend.